test/shimmed.js

```diff
@@ -32,6 +32,18 @@ test('shimmed', function (t) {
 		st.end();
 	});
 
+	t.test('hasOwnProperty', function (st) {
+		const ownProperties = ['for', 'keyFor', 'asyncIterator', 'hasInstance', 'isConcatSpreadable', 'iterator', 'match', 'matchAll', 'replace', 'search', 'species', 'split', 'toPrimitive', 'toStringTag', 'unscopables', 'observable'];
```

Comment:
```suggestion
		var ownProperties = ['for', 'keyFor', 'asyncIterator', 'hasInstance', 'isConcatSpreadable', 'iterator', 'match', 'matchAll', 'replace', 'search', 'species', 'split', 'toPrimitive', 'toStringTag', 'unscopables', 'observable'];
```

Reply:
also, `observable` is not a thing, unless you've perhaps installed a polyfill for the long-defunct stage 2 Observables proposal